### ISSUE TITLE
Keyword dependency_links is not found

### ISSUE THREAD

**Issue by xoxys** [Madoshakalaka/pipenv-setup#23]
`check` command will fail if `dependency_links` is not present in setup.py. Is it intended that we need an (empty) `dependency_links=[]` in setup.py to get the `ckeck` command passed?

**Comment by Madoshakalaka** [comment]
It's not intended, can I have some more details?
Do you have vcs packages _e.g._ git package in your Pipfile/Pipfile.lock? If so, and you have the package in `install_requires` using the new [PEP 508 syntax](https://stackoverflow.com/a/54794506). This bug might happen.
pipenv-setup does not know the new syntax yet. But it is being implemented.

**Comment by xoxys** [comment]
Nope, no vcs packages in Pipfile. But maybe I miss something but you can take a look at your own. https://github.com/xoxys/docker-tidy/blob/master/Pipfile

**Comment by Madoshakalaka** [comment]
@xoxys thanks, I'll investigate this!

**Comment by xoxys** [comment]
Cool. Thanks for your work!

**Comment by inverse** [comment]
Just to confirm this issue is still present - let me know if I can help
